From 76aae88f6f8042ac707c3edb2593c1da17ee2111 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Tue, 15 Jul 2014 16:20:47 +1000
Subject: [PATCH] metadata: no need to re-run the search when a match has
 already been found

(cherry picked from commit 43785e4137cc3eaf4174fdfa62e15061765ba300)
---
 mythtv/libs/libmythmetadata/metadatadownload.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythmetadata/metadatadownload.cpp b/mythtv/libs/libmythmetadata/metadatadownload.cpp
index a5ee929..3bc7c92 100644
--- a/mythtv/libs/libmythmetadata/metadatadownload.cpp
+++ b/mythtv/libs/libmythmetadata/metadatadownload.cpp
@@ -116,7 +116,7 @@ void MetadataDownload::run()
                 else
                     list = handleVideoUndetermined(lookup);
 
-                if (list.isEmpty())
+                if (list.isEmpty() && lookup->GetStep() != kLookupData)
                     list = handleRecordingGeneric(lookup);
             }
             else if (lookup->GetSubtype() == kProbableMovie)
-- 
1.7.10.2

